packages/terra-clinical-item-collection/src/_CreateListView.jsx

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
 import SingleSelectList from 'terra-list/lib/SingleSelectList';
 import ItemView from 'terra-clinical-item-view';
+import styles from './ItemCollection.scss';
 
 function createListView(rows, listStyles, selectedIndex, handleSelection) {
+  const numberOfDisplays = rows[0].displays.length;
+
   const listContent = rows.map((row, rowIndex) => {
+    const { startAccessory, displays, comment, endAccessory, itemStyles } = row;
+
+    const startAccessoryContent = <div className={styles['list-accessory']}>{startAccessory}</div>;
```

Comment:
So this always adds a div with that 40px width. What if none of the items in the list have an accessory? Should they all still have the empty 40px in front?

Reply:
hmm good point. I made some changes in my last commit that simplified the code that took that checkout. (I knew there was a reason I had it initially)  I'll revert those changes.